web-registry/src/components/organisms/RegistryNav.tsx

```diff
@@ -6,11 +6,19 @@ import useMediaQuery from '@mui/material/useMediaQuery';
 import { useTheme } from '@mui/styles';
 
 import Header, { HeaderColors } from 'web-components/lib/components/header';
+import {
```

Comment:
most of this is restoring current Nav from `master`. We will just have to re-delete the "legacy" stuff when ready to launch marketplace v1